### PR TITLE
first step minimize init repo

### PR DESCRIPTION
**Pull request by kids-return**
after will continue to improve

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - $HOME/.composer/cache
+
+install:
+  - travis_retry composer self-update && composer --version
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - travis_retry composer install --dev --prefer-dist --no-interaction
+# install php extensions
+  - |
+    pear config-set preferred_state beta
+    printf "\n" | pecl install imagick
```
Comment (samdark) [inline .travis.yml]: Not needed.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    # gmagick is not installed on travis currently
+    #printf "\n" | pecl install gmagick
+# setup database:
+  - mysql -e 'CREATE DATABASE yii2test;';
+# setup application:
+  - |
+    php install.php init/all --config=tests/travis/install-config.php --interactive=0
+    sed -i "s/'cookieValidationKey' => ''/'cookieValidationKey' => 'testkey'/" config/web.php
```
Comment (samdark) [inline .travis.yml]: Not needed. It's a console.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    cd tests/codeception/backend && codecept build
```
Comment (samdark) [inline .travis.yml]: No need for codecetion for it, I think.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,91 @@
-# yii-base-cli
-Command line framework
+<p align="center">
+    <a href="http://www.yiiframework.com/" target="_blank">
+        <img src="https://www.yiiframework.com/files/logo/yii.png" width="400" alt="Yii Framework" />
+    </a>
+    <h1 align="center">Yii Framework Application</h1>
```
Comment (samdark) [inline README.md]: Command Line Application

File: README.md [hunk 5]
```diff
@@ -1,2 +1,91 @@
...
+    <br>
+</p>
+
+This package is [Yii Framework] application best for rapidly creating projects.
+
+The template contains the basic features including user login/logout and a contact page.
```
Comment (samdark) [inline README.md]: Should be adjusted to reflect what's really there.

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    "homepage": "https://github.com/yiisoft/yii-base-web",
+    "license": "BSD-3-Clause",
+    "support": {
+        "source": "https://github.com/yiisoft/yii-base-cli",
+        "issues": "https://github.com/yiisoft/yii-base-cli/issues",
+        "forum": "http://www.yiiframework.com/forum/",
+        "wiki": "http://www.yiiframework.com/wiki/",
+        "irc": "irc://irc.freenode.net/yii"
+    },
+    "authors": [
```
Comment (samdark) [inline composer.json]: We can remove it, I think. Makes little sense nowadays with such many contributors.

File: composer.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": ">=7.1.0",
+        "yiisoft/yii-console": "^3.0@dev",
+        "yiisoft/di": "^3.0@dev",
+        "yiisoft/log": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
+        "yiisoft/active-record": "^3.0@dev",
```
Comment (samdark) [inline composer.json]: AR isn't a requirement for command line.

**Review by samdark — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (kids-return) [reply .travis.yml]: I'm going to fix it tomorrow, and I thought at first, make sure it works, and then remove the useless stuff, which should be more efficient, and more people can be involved 🙁

**Comment by kids-return** [comment]
@samdark Please help me review the code again. Thank you 😃

**Comment by samdark** [comment]
Thank you!

**Pull request merged**